### PR TITLE
Meeting notes 2020-03-24

### PR DESCRIPTION
**Pull request by jameslibscie**

### CODE REVIEW

**Comment by jameslibscie** [comment]
@juliangruber I have no idea what you mean, but this somehow did help me realise that I forgot to remove Lisa from the attending list 🤷‍♂

File: 20200324.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    - In progress:
+        - Making multiple designs for creating and browsing follow-up content and selecting which content is the `parent` content
+        - Specs discussions regarding whether parent modules need to be published first and which file formats should be allowed
+        - Research for the Daemon
+        - Glossary for Hypergraph and/or Liberate Science
+    - Up next:
+        - Input validation and error handling on Desktop
+        - Exploratory work on Daemon app
+-   [`@p2pcommons/specs`](https://github.com/p2pcommons/specs)
+    - Discussion surrounding uniqueness wording still ongoing (`0.2.4`)
```
Comment (chartgerink) [inline 20200324.md]: Please add URL.

File: 20200324.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        - content: finish publish/unpublish/delete
+        - welcome: fix name missing when going back
+        - add create content with parent module
+        - prototype visualizing parent relationship
+        - JG gives a presentation of latest changes
+            - JL: Why does the version show as v3?
+                - JG+CH: Should we even show the version number?
+                    - Let's drop it for now
+            - PSS: I might want to rearrange the module list if we're rethinking the metadata?
+            - CH: Should the Workspace contain published work?
```
Comment (chartgerink) [inline 20200324.md]: Please drop 64-65

**Comment by juliangruber** [comment]
> I have no idea what you mean,

I was looking for the gif :P

**Pull request merged**